numerals/templates/language/detail_html.mako

```diff
@@ -32,4 +32,11 @@ ${request.get_datatable('values', h.models.Value, language=ctx).render()}
         </%util:accordion_group>
         % endif
     </div>
+
+    <div class="well well-small">
+      % if ctx.contributor:
+      <b>${_('Contributor:')}</b><br />
+      ${ctx.contributor}
+      % endif
```

Comment:
Does this have to be in a custom template? Wouldn't this be a better fit for a column in datatables?

Reply:
the content is in the most case very long - one could add this via detail button link

Reply:
Yes, having this in `datatables.py` retrieved and displayed with `More` (i.e. `DetailsRowLinkCol`) sounds preferable in my opinion.